어드민 스페이스 응답 형태 및 필드 이름 수정

EllieHyun opened the pull request:
## Summary
- Close #212 

<br>

## Changes 
- https://www.notion.so/mansu31/d2d0f064abf4491a965ca277bb48ad45?pvs=4 위 문서 참고하여 필드 이름 변경, 응답 형태 수정 등 요구사항 반영하였습니다!

<br>

## To Reviewers

<br>

EllieHyun committed:
fix: update store space field name and reservation unit format

- reflect requirements of front-end

heyyjin reviewed (changes requested):
몇 가지 요청사항 남겼는데, 관련한 코드 수정 부탁드리겠습니다!!

EllieHyun committed:
fix: update store space field name and reservation unit format

- use json property to use field name
- change isTemporarilyClosed to isClosedToday
- change storeTableId and storeChairId to idByWeb
- remove wifi

heyyjin reviewed (approved):
반영 감사합니다!
